src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -170,19 +172,82 @@ public override string ToString()
 
         // DEPRECATE / OBSOLETE - this functionality is not used and should be removed in a next major version
```

Comment:
```suggestion
        // DEPRECATE / OBSOLETE - this functionality is not used and should be removed in a next major version
```

should remove this, since we know it is being used